kubernetes/nodecollection.go

```diff
@@ -384,7 +409,7 @@ func checkEndpointConnections(config KubeAgentConfig, client *http.Client, metho
 	if err != nil {
 		return false, err
 	}
-	log.Infof("/stats/summary endpoint available via %s connection? %v", method, ns)
+	log.Infof("/stats/summary endpoint on node [%s] available via %s connection? %v", nodeStatSum, method, ns)
```

Comment:
nodeStatSum is of the form "https://{{ ip }}:{{port}}/stats/summary" so we're repeating the endpoint name twice here